modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -2599,6 +2599,72 @@ test.describe('Manage object entries through View Object Entries', () => {
 		}
 	);
 
+	test('verify that API is invoked only once', async ({
```

Comment:
I think we can rename this to something like: `verify that relationship API is called only once when adding object entry`. what do you think? feel free to suggest as well!